x/globalfee/ante/fee.go

```diff
@@ -28,8 +28,8 @@ import (
 var _ sdk.AnteDecorator = FeeDecorator{}
 
 type FeeDecorator struct {
-	GlobalMinFee    globalfee.ParamSource
```

Comment:
@yaruwangway if your changing code, then this should not have the 'doc' prefix in the PR.